restrictedsessions/middleware.py

```diff
@@ -39,7 +39,11 @@ def process_request(self, request):
                 logout(request)
             else:  # logout(...) flushes the session so ensure it only happens once
                 request.session.flush()
-            logger.warning("Destroyed session due to invalid change of remote host or user agent")
+            logger.warning(
+                "Destroyed session due to invalid change of remote host or user agent",
+                user.username,
```

Comment:
Not all users have a username in case of custom user objects. It would probably be best to base this on https://docs.djangoproject.com/en/1.11/topics/auth/customizing/#django.contrib.auth.models.CustomUser.USERNAME_FIELD